src/plot_api/plot_schema.js

```diff
@@ -64,7 +64,7 @@ exports.get = function() {
     return {
         defs: {
             valObjects: Lib.valObjectMeta,
-            metaKeys: UNDERSCORE_ATTRS.concat(['description', 'role']),
+            metaKeys: UNDERSCORE_ATTRS.concat(['description', 'role', 'editType', 'impliedEdits']),
```

Comment:
With this addition, the schema still works as expected in `plotly.py` - thanks @cldougl for helping me figure out how to test that.

Reply:
(and with the schema already in `dist` we won't need to build it in streambed anymore)

Reply:
I believe we should _still_ generate the schema in streambed, and remove the `*src` key [handler](https://github.com/plotly/plotly.js/blob/11bb997e78128b9f47db2ed392982f82a026e589/src/plot_api/plot_schema.js#L355) from `plot_schema.js`.